website/docs/components/code-editor/partials/guidelines/guidelines.md

```diff
@@ -54,18 +54,18 @@ The secondary actions section supports two optional buttons: the [Copy Button](/
 
 ## Custom actions
 
-!!! Warning
-
-The Code Editor has limited support for dark mode styles. Buttons have pre-defined dark mode styles, but all other components require manual color adjustments until a dark mode theme is released. Please [contact the Design Systems Team](/about/support) for help translating components into dark mode.
-
-!!!
-
 Custom primary actions can be added to the header. Primary actions include those necessary for the user to complete their work.
 
 Here is an example of a custom action to reveal secrets.
 
 ![The Code Editor with the title “CodeEditor title”. The custom yielded section has a “Reveal secrets” toggle.](/assets/components/code-editor/code-editor-primary-yielded-elements.png)
 
+!!! Callout
+
+The Code Editor has limited support for dark mode styles. Buttons have pre-defined dark mode styles, but all other components require manual color adjustments until a dark mode theme is released. Please [contact the Design Systems Team](/about/support) for help translating components into dark mode.
+
+!!!
+
```

Comment:
Would you want to move this to be under `## Custom actions`? It looks a little strange here, and I thought we were aiming to put callouts like this directly under a header vs. mixed in together with more general content.

Reply:
@andgen404 I'm noticing a number of Callout banners not being placed below headers. I had thought that we were trying to update all the banners to normally put them directly below a header vs. mixed in with other content. Does that not apply to the Callout banner though?

If I was incorrect in my assumption on desired banner placement, I'd like to know so that I don't waste your time calling out things incorrectly.